Source/IntegrationTests/TestFiles/LitTests/LitTest/triggers/splitting-triggers-recovers-expressivity.dfy.expect

```diff
@@ -6,18 +6,20 @@ splitting-triggers-recovers-expressivity.dfy(26,10): Info: Selected triggers: {Q
  Rejected triggers: {P(i)} (may loop with "P(i + 1)")
 splitting-triggers-recovers-expressivity.dfy(33,11): Info: Selected triggers: {Q(j)}
  Rejected triggers: {P(j)} (may loop with "P(j + 1)")
-splitting-triggers-recovers-expressivity.dfy(44,10): Info: For expression "P(i) || !Q(i)":
-   Selected triggers:
-     {Q(i)}, {P(i)}
-splitting-triggers-recovers-expressivity.dfy(44,10): Info: For expression "P(i + 1)":
-   Selected triggers: {Q(i)}
-   Rejected triggers: {P(i)} (may loop with "P(i + 1)")
-splitting-triggers-recovers-expressivity.dfy(49,11): Info: For expression "j >= 0 ==> P(j)":
-   Selected triggers:
-     {Q(j)}, {P(j)}
-splitting-triggers-recovers-expressivity.dfy(49,11): Info: For expression "j >= 0 ==> Q(j) ==> P(j + 1)":
-   Selected triggers: {Q(j)}
-   Rejected triggers: {P(j)} (may loop with "P(j + 1)")
+splitting-triggers-recovers-expressivity.dfy(44,10): Info: Quantifier was split into 2 parts. Better verification performance and error reporting may be obtained by splitting the quantifier in source. For more information, see the section quantifier instantiation rules in the reference manual.
+splitting-triggers-recovers-expressivity.dfy(44,10): Info: Part #0 is 'P(i) || !Q(i)'
+ Selected triggers:
+   {Q(i)}, {P(i)}
+splitting-triggers-recovers-expressivity.dfy(44,10): Info: Part #1 is 'P(i + 1)'
+ Selected triggers: {Q(i)}
+ Rejected triggers: {P(i)} (may loop with "P(i + 1)")
+splitting-triggers-recovers-expressivity.dfy(49,11): Info: Quantifier was split into 2 parts. Better verification performance and error reporting may be obtained by splitting the quantifier in source. For more information, see the section quantifier instantiation rules in the reference manual.
```

Comment:
I'm not sure I understood with the documentation what it means by "splitting the quantifier in source". You mean, like the exists quantifier example I give below?

Reply:
I mean that you would manually perform the split so Dafny does not have to